verticaltabs.js

```diff
@@ -736,12 +736,22 @@ VerticalTabs.prototype = {
     search_icon.addEventListener('click', function (e) {
       find_input.focus();
     });
-    find_input.addEventListener('input', this.filtertabs.bind(this));
+    find_input.addEventListener('input', () => {
+      window.VerticalTabs.search_engaged = true;
+      this.filtertabs.call(this);
+    });
     this.window.addEventListener('keyup', (e) => {
       if(e.keyCode === 27) {
         this.clearFind();
       }
     });
+    find_input.onfocus = () => { this.sendPing('tab_center_search_focus', window);};
+    find_input.onblur = () => {
+      let details = {search_engaged: window.VerticalTabs.search_engaged};
+      this.sendPing('tab_center_search_blur', window, details);
+      window.VerticalTabs.search_engaged = false;
+    };
+    this.search_engaged = false;
```

Comment:
Why are we sometimes calling `this.search_engaged` and other times `window.VerticalTabs.search_engaged`?  Shouldn't they be the same thing?